armory_compat/config_template.json

```diff
@@ -44,7 +45,7 @@
       "name": "ImageClassificationTask"
   },
   "sysconfig": {
-      "docker_image": "twosixarmory/tf2:0.13.1",
+      "docker_image": "twosixarmory/${FRAMEWORK}:0.13.1",
```

Comment:
Instead of fixing this to a particular armory version (0.13.1) in multiple places, how about setting this to `${VERSION}`, and then in `run_armory.sh` below, pass it into the config template?

You can get the version in bash by doing:
```
VERSION=$(armory --version)
```

That would reduce the cost of future updates.

Reply:
Good idea -- this has been updated to remove the hardcode of version.